ansible_collections/arista/avd/examples/cv-pathfinder/ansible.cfg

```diff
@@ -0,0 +1,7 @@
+[defaults]
```

Comment:
the material folder is showing up with ansible.cfg and build.yml. Should this be cleaned up?

Reply:
yes